nucliadb/nucliadb/writer/tus/__init__.py

```diff
@@ -88,6 +88,9 @@ async def initialize():
 
         DRIVER = TusStorageDriver(backend=storage_backend, manager=storage_manager)
 
+    elif storage_settings.file_backend == FileBackendConfig.notset:
+        raise AttributeError("FILE_BACKEND env variable not configured")
```

Comment:
`ConfigurationError` instead? to be more explicit

Reply:
Yes!